packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -243,12 +243,20 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 			}
 		});
 
+		const immediatelyRefreshLatestSummaryAck =
+			this.mc.config.getBoolean("Fluid.Summarizer.immediatelyRefreshLatestSummaryAck") ??
+			false;
```

Comment:
@agarwal-navin Should this flag be on by default? I ran it on by default for all our e2e tests, and they all pass.

Reply:
Yeah, I think we can make this the default.